app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/FormUtils.scala

```diff
@@ -60,4 +61,23 @@ object FormUtils {
         .find(_._1 === value)
         .fold(Map.empty[String, String]) { case (_, i) => Map(key -> i.toString) }
   }
+
+  def checkBoxAssetTypeFormFormatter: Formatter[AssetType] = new Formatter[AssetType] {
```

Comment:
this is specific to the multiple disposals asset type questions for non uk residents - it should probably go into the companion object of the MultipleDisposalsTriageController. FormUtils is for more general form stuff.